docs_src/examples/app-service-examples/camera-management/Ch-CameraManagement.md

```diff
@@ -0,0 +1,417 @@
+
+```bash
+sudo apt install build-essential
+```
+
+## Steps for running this example:
+
+### 1. Start the EdgeX Core Services and Device Services.
+
+1. Clone `edgex-compose` from github.com.
+    ```shell 
+    git clone https://github.com/edgexfoundry/edgex-compose.git
+    ```  
+
+1. Navigate to the `edgex-compose` directory:
+    ```bash
+    cd edgex-compose
+    ```
+
+1. Checkout the latest compatible release branch
+
+    !!! note
+        The `minnesota` branch is the latest stable branch at the time of this writing.
+
+    ```shell
+    git checkout minnesota
+    ```
+
+1. Navigate to the `compose-builder` subdirectory:
+    ```bash
+    cd compose-builder/
+    ```
+
+1. (Optional) Update the `add-device-usb-camera.yml` file:
+
+    !!! note
+        This step is only required if you plan on using USB cameras.
+
+    a. Add the rtsp server hostname environment variable to the `device-usb-camera` service, where `your-local-ip-address` is the ip address of the machine running the `device-usb-camera` service.
+
+    !!! example - "Snippet from `add-device-usb-camera.yml`"
+        ```yml
+        services:
+          device-usb-camera:
+            environment:
+              DRIVER_RTSPSERVERHOSTNAME: "your-local-ip-address"
+        ```
+
+    b. Under the `ports` section, find the entry for port 8554 and change the host_ip from `127.0.0.1` to either `0.0.0.0` or the ip address you put in the previous step.
+
+1. Configure [device-mqtt] service to send [Edge Video Analytics Microservice][evam] inference results into Edgex via MQTT
+
+    a. Copy the entire [evam-mqtt-edgex](https://github.com/edgexfoundry/edgex-examples/tree/{{version}}/application-services/custom/camera-management/edge-video-analytics/evam-mqtt-edgex) folder into `edgex-compose/compose-builder` directory.
+
+    b. Copy this information into the [add-device-mqtt.yml](https://github.com/edgexfoundry/edgex-compose/blob/{{version}}/compose-builder/add-device-mqtt.yml) file in the `edgex-compose/compose-builder` directory.
+
+    !!! example - "Snippet from add-device-mqtt.yml"
+        ```yaml
+        services:
+        device-mqtt:
+          ...
+          environment:
+            DEVICE_DEVICESDIR: /evam-mqtt-edgex/devices
+            DEVICE_PROFILESDIR: /evam-mqtt-edgex/profiles
+            MQTTBROKERINFO_INCOMINGTOPIC: "incoming/data/#"
+            MQTTBROKERINFO_USETOPICLEVELS: "true"
+          volumes:
+            # example: - /home/github.com/edgexfoundry/edgex-compose/compose-builder/evam-mqtt-edgex:/evam-mqtt-edgex
+            - <add-absolute-path-of-your-edgex-compose-builder-here-example-above>/evam-mqtt-edgex:/evam-mqtt-edgex
+        ```
+    c. Copy this information into the [add-mqtt-broker.yml](https://github.com/edgexfoundry/edgex-compose/blob/{{version}}/compose-builder/add-mqtt-broker.yml) file in the `edgex-compose/compose-builder` directory.
+
+    !!! example - "Snippet from add-device-mqtt.yml"
+        ```yaml
+        mqtt-broker:
```

Comment:
I think it would be more helpful if you add `...` like the previous yml file for this too so that the user who is not familiar with docker will not just copy only this info. Add `...` for the ports section too.